synapseformation/client.py

```diff
@@ -59,6 +59,21 @@ def _create_synapse_resources(config: dict, creation_cls: SynapseCreation,
         entity = creation_cls.get_or_create_folder(
             name=config['name'], parentId=parentid
         )
+    elif isinstance(config, dict) and config.get('type') == "Team":
+        team = creation_cls.get_or_create_team(
+            name=config['name'], description=config['description'],
+            canPublicJoin=config['can_public_join']
+        )
+        if config.get("invitations") is not None:
+            # TODO: investigate nested forloop.  This is ugly
```

Comment:
This is still really readable to me.  Perhaps there's an itertools way of doing it, but I'm not aware of it!

*did you mean to remove the #TODO?